### PR TITLE
Gimbal device tester improvements

### PR DESCRIPTION
**Pull request by julianoes**
A collection of improvements for the tester. For more info check commits.

### CODE REVIEW

File: examples/gimbal_device_tester/gimbal_device_tester.cpp [hunk 1]
```diff
@@ -270,24 +270,26 @@ class Tester {
...
-        std::this_thread::sleep_for(std::chrono::seconds(2));
-        std::cout << "DONE\n";
+        // Let's not yaw for now because it is confusing for a gimbal.
+        // std::cout << test_prefix << "Yaw 20 degrees to the right... ";
+        //_attitude_data.change_vehicle_attitude([](AttitudeData::VehicleAttitude& vehicle_attitude)
+        //{
+        //    vehicle_attitude.yaw_deg = 20.0f;
+        //});
+        // std::this_thread::sleep_for(std::chrono::seconds(25));
+        // std::cout << "DONE\n";
```
Comment (JonasVautherin) [inline examples/gimbal_device_tester/gimbal_device_tester.cpp:280]:
```suggestion
```

Comment (julianoes) [reply examples/gimbal_device_tester/gimbal_device_tester.cpp:280]: Not this time :smile:

Comment (JonasVautherin) [reply examples/gimbal_device_tester/gimbal_device_tester.cpp:280]: Do we need to keep this unused code? :thinking:

**Review by JonasVautherin — APPROVED** [review]

File: examples/gimbal_device_tester/gimbal_device_tester.cpp (hunk 1 above)
Comment (julianoes) [reply examples/gimbal_device_tester/gimbal_device_tester.cpp:280]: Yes, so that a reader sees how it would work.

Comment (julianoes) [reply examples/gimbal_device_tester/gimbal_device_tester.cpp:280]: Or if I use it against SITL I would use it.

**Pull request merged**